backend/apps/gamification/services.py

```
@@ -0,0 +1,25 @@
+from datetime import date, timedelta
+
+from .models import Streak
+
+
+class StreakService:
+    @staticmethod
+    def update_streak(user):
+        streak, created = Streak.objects.get_or_create(user=user)
```

Ruff (0.15.20)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)